xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -409,9 +581,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Read the next JSON token from the input source.</summary>
+        <returns><see langword="true" /> if the token was read successfully; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.Text.Json.JsonReaderException">
+            An invalid JSON token is encountered according to the JSON RFC.
```

Comment:
This seems technically untrue.  The RFCs and ECMA-404 all say comments don't exist (therefore are invalid) or trailing commas are disallowed, et cetera.

I'd just go with `An invalid JSON token is encountered.` for this statement.